hazelcast-jet-core/src/main/java/com/hazelcast/jet/Processors.java

```diff
@@ -260,6 +263,38 @@ public static ProcessorSupplier readSocket(@Nonnull String host, int port) {
     }
 
     /**
+     * Convenience for {@link #writeFile(String, Charset, boolean, boolean)}, with
+     * UTF-8 charset, overwriting the target file and no early file buffer flush.
+     */
+    @Nonnull
+    public static ProcessorSupplier writeFile(@Nonnull String file) {
+        return writeFile(file, null, false, false);
+    }
+
+    /**
+     * Returns a supplier of processor, that writes all items to a local file on
+     * each member. {@code item.toString()} is written to the file, followed by
+     * {@code '\n'} (both on linux and Windows).
+     * <p>
+     * The same file must be available for writing on all nodes. The file on
+     * each node will contain part of the data processed on that member.
+     * <p>
+     *  The vertex should have {@link Vertex#localParallelism(int) local parallelism} of 1.
```

Comment:
imagine you have a big machine which have a lot of memory, you can't have a single JVM which uses all the memory but you start multiple JVM instances so that you don't suffer from big GC pauses. so there are uses-cases which you start 2 nodes in a single machine